example/android/app/build.gradle

```diff
@@ -120,6 +120,7 @@ android {
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
         versionName "1.0"
+        multiDexEnabled true
```

Comment:
Did you need to enable multi-dex? We usually don't do it on our sample apps.

Reply:
Yes I think. I have this error when the multi-dex is disabled:

`AGPBI: {"kind":"error","text":"Cannot fit requested classes in a single dex file (# fields: 66055 > 65536)","sources":[{}],"tool":"D8"}
com.android.builder.dexing.DexArchiveMergerException: Error while merging dex archives: 
The number of method references in a .dex file cannot exceed 64K.`